src/main/python/sweep_table_view.py

```diff
@@ -1,17 +1,35 @@
-from PyQt5.QtWidgets import QTableView, QDialog, QGridLayout
+from typing import Optional
+
+from PyQt5.QtWidgets import QTableView, QDialog, QGridLayout, QWidget
 from PyQt5.QtCore import QModelIndex
 
 from pyqtgraph import mkPen, PlotWidget
 
 from delegates import SvgDelegate, ComboBoxDelegate
 from sweep_table_model import SweepTableModel
+from sweep_plotter import PopupPlotter
```

Comment:
I don't see this being used, did I miss it somewhere?

Reply:
answer is: in a previous version I was using it for a type annotation and forgot to remove it. will remove.